content/docs/setup/kubernetes/sidecar-injection.md

```diff
@@ -59,25 +49,21 @@ Automatic injection injects at pod creation time. The controller resource is
 unmodified. Sidecars can be updated selectively by manually deleting a pods or
 systematically with a deployment rolling update.
 
-Manual and automatic injection use the same templated configuration. Automatic
-injection loads the configuration from the `istio-sidecar-injector` ConfigMap in the
-`istio-system` namespace. Manual injection can load from a local file or from
-the ConfigMap.
+Manual and automatic injection both use the configuration from the 
+`istio-sidecar-injector` and `istio` ConfigMaps in the `istio-system` namespace. 
+Manual injection can also optionally load configuration from local files.
 
 ### Manual sidecar injection
 
-Use the built-in defaults template and dynamically fetch the mesh
-configuration from the `istio` ConfigMap. Additional parameter overrides
-are available via flags (see `istioctl kube-inject --help`).
+Inject the sidecar into the deployment using the in-cluster configuration.
 
 ```command
-$ kubectl apply -f <(istioctl kube-inject -f @samples/sleep/sleep.yaml@)
+$ istioctl kube-inject -f @samples/sleep/sleep.yaml@ | kubectl apply -f -
 ```
 
-`kube-inject` can also be run without access to a running Kubernetes
-cluster. Create local copies of the injection and mesh configmap.
+Alternatively, inject using local copies of the configuration.
 
-> The `istioctl kube-inject` operation may not be repeated on the output
+> NOTE: The `istioctl kube-inject` operation may not be repeated on the output
```

Comment:
I believe @geeknoid was attempting to remove the usage of `NOTE: ` from the codebase.  Martin can you confirm?

For NOTEs, can you use something like this:
```
> The `istioctl kube-inject`
```

@geeknoid do you have a preference here as to handling the notes?  Seems like an icon would be warranted in the future - had a quick look at how to implement that - seems straightforward with exception of the icon itself.

Reply:
Yes, please just use the > syntax for notes, without the word NOTE:

You can insert one of two icons at the moment:

  > {{< warning_icon >}} This is a warning note

  > {{< idea_icon >}} This is a neato idea to try out